### PR TITLE
Feat android response models

### PR DESCRIPTION
**Pull request by abnegate**
Add Android response models
Update dependencies
Update to Java 11
Target Android 12

### CODE REVIEW

File: tests/SDKTest.php [hunk 1]
```diff
@@ -77,7 +77,7 @@ class SDKTest extends TestCase
                 'chmod +x tests/sdks/android/gradlew',
             ],
             'envs' => [
-                'java-8' => 'docker run --rm -v $(pwd):/app -w /app/tests/sdks/android alvrme/alpine-android:latest-jdk8 sh -c "./gradlew :library:testReleaseUnitTest -q && cat library/result.txt"',
+                'java-11' => 'docker run --rm -v $(pwd):/app -w /app/tests/sdks/android alvrme/alpine-android:latest-jdk11 sh -c "./gradlew :library:testReleaseUnitTest -q && cat library/result.txt"',
```
[the remark this answers is not in the document]
Comment (abnegate) [reply tests/SDKTest.php:80]: The library can still be used on Java 8 - we are just compiling with 11 (stable Android Studio does now too)

**Comment by abnegate** [comment]
@lohanidamodar I've pushed a snapshot on [dev](https://github.com/appwrite/sdk-for-android/tree/dev) 🙂

File: templates/android/build.gradle.twig [hunk 2]
```diff
@@ -2,15 +2,15 @@ apply plugin: 'io.github.gradle-nexus.publish-plugin'
 
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
-    ext.kotlin_version = "1.4.31"
-    version '{{ sdk.version }}'
+    ext.kotlin_version = "1.5.31"
+    version "{{ sdk.version }}"
```
Comment (christyjacob4) [inline templates/android/build.gradle.twig:6]: Are double quotes and single quotes treated similarly ?

**Review by christyjacob4 — CHANGES_REQUESTED** [review]

File: templates/android/build.gradle.twig (hunk 2 above)
Comment (abnegate) [reply templates/android/build.gradle.twig:6]: Yep they are the same except allow for interpolation. I changed for consistency but I see there's other I missed anyway

**Review by lohanidamodar — APPROVED** [review]

**Review by christyjacob4 — APPROVED** [review]

**Review by TorstenDittmann — APPROVED** [review]

**Pull request merged**